src/Sushi.Mediakiwi/Controllers/Supporting/IController.cs

```diff
@@ -5,7 +5,5 @@ namespace Sushi.Mediakiwi.Controllers
 {
     public interface IController
     {
-        bool IsAuthenticationRequired { get; set; }
-        //Task<string> CompleteAsync(HttpContext context);
     }
```

Comment:
An empty interface, can this be deleted ?

Reply:
Don't know why this was removed. Added new version.